test/nomad_cluster_ssh_test.go

```diff
@@ -0,0 +1,13 @@
+package test
+
+import "testing"
+
+func TestNomadClusterSSHAccess(t *testing.T) {
```

Comment:
And if we're reverting the examples, we probably need to revert this new test code too (sounds like you hit a Terraform bug with it anyway).

Reply:
I would suggest to keep the positive test. This will provide feedback if an SSH connection is possible. This can be done without being affected by the bug.

Reply:
Roger